e2e-test/trace/src/integrationTest/resources/raw-span-e2e-pipeline-latest-release.yml

```diff
@@ -15,7 +11,7 @@ raw-pipeline:
     pipeline:
       name: "entry-pipeline"
   processor:
-    - otel_trace_raw_prepper:
+    - otel_trace_raw:
```

Comment:
If I remember correctly, current latest release (1.5.1) still needs it to be `otel_trace_raw_prepper:`.

Reply:
I disabled this workflow in GitHub. We should enable it back when 2.0 is released or it'll fail. 
So we'll be doing compatibility test between 2.0 and main branch (any other 2.x release) in future. We won't use `otel_trace_raw_prepper` any more.